Scripts/RTS/AtlasWeight.cs

```diff
@@ -0,0 +1,13 @@
+namespace RTS;
+using Godot;
+
+public class AtlasWeight
+{
+    public AtlasWeight(Vector2I tilePosition, float weight)
+    {
+        this.TilePosition = tilePosition;
+        this.Weight = weight;
+    }
+    public Vector2I TilePosition;
```

Comment:
public fields should be public properties. So this should be `public Vector2I TilePosition { get; }` and `public float Weight { get; }`

Reply:
Also please put the properties above the constructor.